test/kpod.bats

```diff
@@ -2,8 +2,48 @@
 
 load helpers
 
+ROOT="$TESTDIR/crio"
+RUNROOT="$TESTDIR/crio-run"
+KPOD_OPTIONS="--root $ROOT --runroot $RUNROOT --storage-driver vfs"
```

Comment:
When I drop this, all my tests fail with something about overlay not being supported.